rst_files/ifp.rst

```diff
@@ -493,7 +493,7 @@ Setup
         return V, c
     end
 
-Both ``bellman_operator`` and ``coleman_operator`` use linear interpolation along the asset grid to approximate the value and consumption functions
+Both ``T`` and ``coleman_operator`` use linear interpolation along the asset grid to approximate the value and consumption functions
```

Comment:
We probably shouldn't get rid of `bellman_operator` and leave `coleman_operator`. In the lecture we use _K_ to denote that operator (see the `Col90` ref); could you make that shift?

Reply:
Of course. First, I tried to change only bellman_operator, and now I am going through each lecture notes by the order in http://compare-lectures.quantecon.org/jl/index.html.

Reply:
Gotcha. There's maybe something to be said for doing this per-lecture (otherwise, how do you remember which issues remain in which lecture?) But I'll let you handle the workflow.